metricshub-jawk-extension/src/main/java/org/sentrysoftware/metricshub/extension/jawk/MetricsHubExtensionForJawk.java

```diff
@@ -278,7 +279,13 @@ private String executeWmiRequest(final Object[] args) {
 	private String executeSource(final Source source) {
 		final SourceTable sourceTableResult = source.accept(sourceProcessor);
 
-		return sourceTableResult != null && !sourceTableResult.isEmpty() ? sourceTableResult.getRawData() : "";
+		if (sourceTableResult != null && !sourceTableResult.isEmpty()) {

```

Comment:
Note for future: it would be good if *SourceTable* handled the conversion between raw and table formats automatically:

* *getRawData()* would always return a String, (if raw string wasn't available, convert the table value to string)
* *getTable()* would always return a table (if the table is not available, convert the string to a table)

For further future, *SourceTable* would be able to store different content types: string, table, JSON, etc. and convert from 1 format to another automatically (or as specified by the connector).